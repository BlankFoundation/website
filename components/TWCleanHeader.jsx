import {
  TWHorizontalSpread
} from '.'

const TWCleanHeader = ({ className, children }) => {
  return (
    <div className={`${className} p-6 bg-white text-black leading-10`}>
      <TWHorizontalSpread className='sm:px-12 md:px-24'>
        {children}
      </TWHorizontalSpread>
    </div>
  );
}

TWCleanHeader.defaultProps = {
  children: ['child1', 'child2', 'child3'].map(
    (child, index) => <div key={'child-' + index}>{child}</div>
  ),
  className: ""
}

export default TWCleanHeader;