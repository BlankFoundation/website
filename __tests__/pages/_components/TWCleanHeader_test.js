import React from 'react';
import TestRenderer from 'react-test-renderer';
import TWCleanHeader from '../../../pages/components/TWCleanHeader';

describe("TWCleanHeader", () => {
  it('renders correctly', () => {
    const rendered = TestRenderer.create(<TWCleanHeader />).toJSON();
    expect(rendered).toMatchSnapshot()
  });
});