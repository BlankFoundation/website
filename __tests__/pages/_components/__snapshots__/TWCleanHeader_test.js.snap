// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TWCleanHeader renders correctly 1`] = `
<div
  className=" p-6 bg-white text-black leading-10"
>
  <div
    className="px-24 flex flex-row justify-between"
  >
    <div>
      child1
    </div>
    <div>
      child2
    </div>
    <div>
      child3
    </div>
  </div>
</div>
`;
